openbas-api/src/main/java/io/openbas/migration/V3_202409191705__Add_table_inject_dependencies.java

```diff
@@ -0,0 +1,49 @@
+package io.openbas.migration;
+
+import io.openbas.database.model.Variable;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.stereotype.Component;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+@Component
+public class V3_202409191705__Add_table_inject_dependencies extends BaseJavaMigration {
```

Comment:
This is easier for devs as we had conflict between versions everytime we switched from a branch to another. We had a team discussion about it, I thought you were there. I did it on this PR for testing if it works well or not. Do you think we should talk about it again ?

Reply:
It's ok for me if everyone is aligned with this new process, I asked because I don't see it in the Guidelines.

Using timestamps requires special attention: there will no longer the same version number twice, but if a PR with a migration of 10/09 is merged before a PR containing a migration of 08/09. . do we need to change the migration date from 09/08 to 09/11 to be sure that it is taken into account ?

Reply:
As seen together, we'll move to this new format during an ulterior PR